backend/src/main/java/com/woowacourse/naepyeon/service/RollingpaperService.java

```diff
@@ -107,6 +111,21 @@ rollingpaper, findRollingpaperAddresseeNickname(rollingpaper))
         return new RollingpapersResponseDto(rollingpaperPreviewResponseDtos);
     }
 
+    private List<Rollingpaper> filterRollingpapers(final Long teamId, final String order, final String filter) {
+        final List<Rollingpaper> rollingpapers = rollingpaperRepository.findByTeamIdAndOrder(teamId, order);
+        // filter 값이 null일 경우
+        if (!StringUtils.hasText(filter)) {
+            return rollingpapers;
+        }
+        final Optional<Recipient> recipient = Recipient.from(filter);
+        if (recipient.isEmpty()) {
+            return rollingpapers;   // filter 값이 member, team 외의 값일 경우 전부 리턴
+        }
+        return rollingpapers.stream()
+                .filter(rollingpaper -> rollingpaper.checkSameRecipient(recipient.get()))
+                .collect(Collectors.toUnmodifiableList());
+    }
```

Comment:
```suggestion
        final Recipient recipient = Recipient.from(filter);
        if (recipient.equals(Recipient.NONE)) {
            return rollingpapers; 
        }
        return rollingpapers.stream()
                .filter(rollingpaper -> rollingpaper.checkSameRecipient(recipient))
                .collect(Collectors.toUnmodifiableList());
    }
```
filter가 null이거나 다른 값이 들어왔는가에 대한 부분을 
Recipient에 NONE타입을 추가해서 Recipient.from에서 처리해주는 방법이 좋을 것 같습니다.
따라서 null이거나 다른 값이 들어왔을때 
```java
    public static Recipient from(final String filter) {
        return Arrays.stream(values())
                .filter(value -> value.recipient.equalsIgnoreCase(filter))
                .findFirst()
                .orElse(NONE);
```
이런 식으로 처리한다면 해당 로직에 대한 고민이 대부분 해결되지 않을까요?

Reply:
Recipient가 도메인 enum이라 NONE으로 할 경우 해당 롤링페이퍼의 수신인 타입이 없는 것이라는 느낌이 강하게 됩니다. ALL이라 해도 같은 문제가 발생하구요.

아예 FILTER라는 Enum을 만드는 것도 생각중인데 좀 더 고민해봐야겠네요

Reply:
저는 개인적으론 필터 타입을 만드는 것보단 현재 있는 걸 가지고 로직으로 풀어내는게 더 합리적으로 보입니다!